Competition2017/src/competition/subsystems/autonomous/ShootFromHopperCommandGroup.java

```diff
@@ -0,0 +1,75 @@
+package competition.subsystems.autonomous;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.drive.commands.DriveForDistanceAtHeadingCommand;
+import competition.subsystems.drive.commands.RotateToHeadingCommand;
+import competition.subsystems.shift.ShiftSubsystem.Gear;
+import competition.subsystems.shift.commands.ShiftGearCommand;
+import competition.subsystems.shoot_fuel.LeftShootFuelAndAgitateCommandGroup;
+import competition.subsystems.shooter_wheel.ShooterWheelSubsystem.TypicalShootingPosition;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+import xbot.common.subsystems.pose.commands.SetRobotHeadingCommand;
+
+public class ShootFromHopperCommandGroup extends CommandGroup {
+    private final DoubleProperty redAllianceStartingHeading;
+    private final DoubleProperty blueAllianceStartingHeading;
+
+    private final DoubleProperty redHeadingToHopper;
+    private final DoubleProperty blueHeadingToHopper;
+
+    private final DoubleProperty distanceToTravelToHopper;
+    
+    protected final SetRobotHeadingCommand setInitialHeading;
+    protected final DriveForDistanceAtHeadingCommand activateHopper;
+    
+    @Inject
+    public ShootFromHopperCommandGroup(
```

Comment:
This command goes to the hopper, collects and shoots. I'm trying to differentiate this from "shoot and then align with hopper." What do you think?